docs/products/flink/concepts/kafka-connectors.rst

```diff
@@ -1,5 +1,5 @@
-Standard and upsert Apache Kafka® connectors
-============================================
+Upsert connectors for Apache Kafka®
```

Comment:
This title was both standard (insert only) and Upsert... why are we changing it to be upsert only?

Reply:
Changed this to Standard and upsert connectors.